pkgs/applications/misc/zathura/wrapper.nix

```diff
@@ -1,17 +1,13 @@
 { symlinkJoin, lib, makeWrapper, zathura_core, plugins ? [] }:
-
-let
-  pluginsPath = lib.makeSearchPath "lib/zathura" plugins;
-
-in symlinkJoin {
+symlinkJoin {
   name = "zathura-with-plugins-${zathura_core.version}";
 
-  paths = [ zathura_core ];
+  paths = [ zathura_core ] ++ plugins;
 
   buildInputs = [ makeWrapper ];
 
   postBuild = ''
-    wrapProgram $out/bin/zathura --add-flags --plugins-dir=${pluginsPath}
+    wrapProgram $out/bin/zathura --add-flags --plugins-dir="''${out}/lib/zathura"
```

Comment:
I don't know anything about zathura, but is the wrapper still needed now that plugins are in an "usual" location ?

Reply:
I tried to remove it but without it, zathura tries to load plugins from zathura-core store, while symlinkJoin creates a new folder zathura-with-plugins.